index.html

```diff
@@ -0,0 +1,276 @@
+              <img
+                class="main-program__card__icon"
+                src="assets/icons/talks-icon.png"
+                alt=""
+              />
+              <h3 class="main_program__card__header">Talks</h3>
+              <p class="main-program__card__description">
+                Listen to inspiring speakers from various countries about their
+                experience in African Education.
+              </p>
+            </li>
+
+            <li class="main-program__card">
+              <img
+                class="main-program__card__icon"
+                src="assets/icons/exhibition-icon.png"
+                alt=""
+              />
+              <h3 class="main_program__card__header">Exhibition</h3>
+              <p class="main-program__card__description">
+                Appreciate cutting edge approaches currently being experimented
+                with all thought the African continent.
+              </p>
+            </li>
+
+            <li class="main-program__card">
+              <img
+                class="main-program__card__icon"
+                src="assets/icons/forum-icon.png"
+                alt=""
+              />
+              <h3 class="main_program__card__header">Forum</h3>
+              <p class="main-program__card__description">
+                Have the time to share your thoughts and opinions with experts
+                for each topic.
+              </p>
+            </li>
+
+            <li class="main-program__card">
+              <img
+                class="main-program__card__icon"
+                src="assets/icons/workshop-icon.png"
+                alt=""
+              />
+              <h3 class="main_program__card__header">Workshop</h3>
+              <p class="main-program__card__description">
+                Experience first hand how lessons are delivered to students,
+                rather than simply listening talks about it.
+              </p>
+            </li>
+
+            <li class="main-program__card">
+              <img
+                class="main-program__card__icon"
+                src="assets/icons/ignite-icon.png"
+                alt=""
+              />
+              <h3 class="main_program__card__header">Ignite</h3>
+              <p class="main-program__card__description">
+                Get opportunities to network with affiliates and experts around
+                the continent.
+              </p>
+            </li>
+          </ul>
+          <button class="primary-btn join-btn">
+            Join African Education Summit 2023
+          </button>
+        </div>
+      </section>
+      <section class="speakers">
+        <div class="container">
+          <h2 class="speakers__headline small-underline">Featured Speakers</h2>
+          <div
+            id="speakers__content"
+            class="speakers__content"
+            data-expanded="false"
+          ></div>
+        </div>
+      </section>
```

Comment:
- [ ] The `more` button is not working, kindly make sure to fix it so that when the user clicks the button all the speakers display on the page  

![Screenshot_20230529_233635](https://github.com/henokkhm/microverse-module-1-capstone-project/assets/25227022/034227a5-88de-4de7-9b74-2e9aea0c9c57)

Reply:
Hi @Ismailco,
Both the **More** button and the **Less** button work properly, even on the live demo. I just double-checked. So can you please check again?

Reply:
I now realize what the issue was. In my JS script, I checked only if the button was clicked. That is, **I failed to check if the text inside the button or the caret icon was clicked**. Hence, the button did what the user expected only if it was clicked in the area shaded in blue. 
![image](https://github.com/henokkhm/microverse-module-1-capstone-project/assets/110997325/2f6380f5-8476-44c5-82b7-eda109a56d28) 
Thank you for finding the bug.